l10n_ch_scan_bvr/wizard/scan_bvr.py

```diff
@@ -485,61 +348,64 @@ def validate_bvr_string(self, cr, uid, ids, context):
         # Explode and check  the BVR Number and structurate it
         #
         data = {}
-        data['bvr_struct'] = self._get_bvr_structurated(
-            bvr_data.bvr_string)
+        data['bvr_struct'] = self._get_bvr_structurated(self.bvr_string)
+        partner_bank_model = self.env['res.partner.bank']
+        partner_bank = False
         # We will now search the account linked with this BVR
         if data['bvr_struct']['domain'] == 'name':
             domain = [('acc_number', '=', data['bvr_struct']['beneficiaire'])]
```

Comment:
I agree with you, so someone should fix it in other versions too, see f.e.:
https://github.com/OCA/l10n-switzerland/blob/7.0/l10n_ch_scan_bvr/wizard/scan_bvr.py#L488
https://github.com/OCA/l10n-switzerland/blob/8.0/l10n_ch_scan_bvr/wizard/scan_bvr.py#L343